index.bs

```diff
@@ -142,48 +142,52 @@ To <dfn>construct a <a>Geolocation Sensor</a> object</dfn> the user agent must i
 ### GeolocationSensor.latitude ### {#geolocationsensor-latitude}
 
 The {{GeolocationSensor/latitude!!attribute}} attribute of the {{GeolocationSensor}} interface
-represents the latitude coordinate of the [=geolocation=] in decimal degrees [[!WGS84]]. This
-attribute returns [=map/value=] for [=latest reading=]["latitude"] [=map/entry=]. Initially, its
-value must be null.
+returns the result of invoking [=get value from latest reading=] with `this` and "latitude" as
+arguments. Initially, its value must be null. It represents the latitude coordinate of the
```

Comment:
> Initially, its value must be null.

Is this needed? It is `nullable`, so, it will be null initially. Similarly, I think, "`Initially, its value must be null`" can be removed from other definitions.

Reply:
Indeed https://w3c.github.io/sensors/#get-value-from-latest-reading returns null if the state is not activated, so we could get rid of these.